js/ui/control/attribution_control.js

```diff
@@ -67,11 +67,11 @@ AttributionControl.prototype = util.inherit(Control, {
             }
             return true;
         });
-
         this._container.innerHTML = attributions.join(' | ');
     },
 
     _updateEditLink: function() {
+        if (!this._editLink) this._editLink = this._container.querySelector('.mapbox-improve-map');
```

Comment:
This is another reason why I had the logic in `_updateAttributions` before. DOM updates happen if new sources are added. Do you think it's preferable to re-assign `this._editLink` on every `mapmove` / `_updateEditLink` call or put assignment into `_updateAttributions`?